recipes/dask-gateway/meta.yaml

```diff
@@ -0,0 +1,112 @@
+
```

Comment:
name is undefined, so nothing is running in any of the CI builds (odd that this isn't an error).

```suggestion
{% set name = "dask-gateway" %}
```
```

Reply:
really unclear how that got lost :confused: